examples/Kaguya_TC.ipynb

```diff
@@ -26,10 +26,10 @@
     "import pvl\n",
     "import spiceypy as spice\n",
     "from ale.drivers import JsonEncoder\n",
-    "from ale.drivers.kaguya_driver import TcPds3Driver\n",
+    "from ale.drivers.kaguya_drivers import KaguyaTcPds3NaifSpiceDriver\n",
     "\n",
-    "# This dynamically registers this plugin with the CSM API.\n",
-    "lib = ctypes.CDLL('/Users/kelvin/repos/usgscsm/build/libusgscsm.dylib')"
+    "# This dynamically registers this plugin with the CSM API. Plugin here is located in Conda environment\n",
+    "lib = ctypes.CDLL('{}/lib/libusgscsm.dylib'.format(os.environ[\"CONDA_PREFIX\"]))"
```

Comment:
This will not work on Linux (.dylib vs. .so) -- is this something we want to make OS-agnostic somehow?

Reply:
This is already codified in knoten. If you import knoten, you don't need this line